readthedocs/rtd_tests/fixtures/spec/v2/schema.yml

```diff
@@ -53,22 +53,30 @@ python:
   # Default: '3'
   version: enum('2', '2.7', '3', '3.3', '3.4', '3.5', '3.6', required=False)
 
+  # Give the virtual environment access to the global site-packages dir
+  # Default: false | project config
+  system_packages: bool(required=False)
+
+  # Installation of packages and requiremens
+  install: list(any(include('python-install-requirements'), include('python-install')), required=False)
+
+python-install-requirements:
   # The path to the requirements file from the root of the project
-  # Default: null | project config
-  requirements: path(required=False)
+  requirements: path()
```

Comment:
No more hacks with `requirements: ''` (empty string) to avoid installing from requirements (we can keep that behaviour for v1)